modules/virt-must-gather-options.adoc

```diff
@@ -41,25 +43,29 @@ Each script is compatible only with certain environment variable combinations.
 
 `/usr/bin/gather --images`:: Collect image, image-stream, and image-stream-tags custom resource information. This script is compatible only with the `PROS` variable.
 
+`/usr/bin/gather --instancetypes`:: Collect InstanceTypes information. This information is not currently collected by default; you can, however, optionally collect it.
```

Comment:
```suggestion
`/usr/bin/gather --instancetypes`:: Collect InstanceTypes information. This information is not currently collected by default. Optionally, you can collect it.
```

Reply:
Up to you, but I think it's sufficient to say that it's not collected by default.
```suggestion
`/usr/bin/gather --instancetypes`:: Collect InstanceTypes information. This information is not currently collected by default.
```